rollup.config.js

```diff
@@ -0,0 +1,26 @@
+import filesize from 'rollup-plugin-filesize';
+import {terser} from 'rollup-plugin-terser';
+
+export default {
+  input: 'index.js',
+  output: {
+    file: 'index.bundled.js',
+    format: 'esm',
+  },
+  onwarn(warning) {
+    if (warning.code !== 'CIRCULAR_DEPENDENCY') {
```

Comment:
@rictic does Closure support circular dependencies?

Reply:
It supports some simple cases I tested, but I don't know how firm their guarantees of correctness are. We might want to test it out